ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/ConversionsSpec.scala

```diff
@@ -93,6 +96,190 @@ class ConversionsSpec extends AnyWordSpec with Matchers {
       val key2 = deduplicationKeyBytesFor(List("alice", "bob"))
       key1 should not be key2
     }
+
+    "encode/decode rejections" should {
+
+      val submitterInfo = DamlSubmitterInfo.newBuilder().build()
+      val now = Instant.now
+
+      "convert rejection to proto models and back to expected grpc code" in {
```

Comment:
What is our strategy for checking (driver-specific afair) details?

Reply:
Nothing really being done for now. We only verify status codes and some messages in a few conformance tests, but nothing really extended